src/index.html

```diff
@@ -17,11 +17,350 @@
+          alt="Photo of product"
+          class="product-card__image"
+        />
+
+        <h2 class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-card__code">Product code: 195434</p>
+
+        <section class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__reviews">Reviews: 5</p>
+        </section>
+
+        <div class="product-card__price">
+          <p class="product-card__price-label">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+
+        <button class="product-card__buy-btn">BUY</button>
+      </div>
+      <div class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Photo of product"
+          class="product-card__image"
+        />
+
+        <h2 class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-card__code">Product code: 195434</p>
+
+        <section class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__reviews">Reviews: 5</p>
+        </section>
+
+        <div class="product-card__price">
+          <p class="product-card__price-label">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+
+        <button class="product-card__buy-btn">BUY</button>
+      </div>
+      <div class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Photo of product"
+          class="product-card__image"
+        />
+
+        <h2 class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-card__code">Product code: 195434</p>
+
+        <section class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product-card__reviews">Reviews: 5</p>
+        </section>
+
+        <div class="product-card__price">
+          <p class="product-card__price-label">Price:</p>
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+
+        <button class="product-card__buy-btn">BUY</button>
+      </div>
```

Comment:
The product cards from line 147 to line 363 are duplicates of the first product card. If this is intentional for layout purposes, consider varying the content (e.g., product name, code, price) to better simulate a real catalog. If not, you may want to remove or modify these duplicates.